sklearn/neural_network/rbm.py

```diff
@@ -73,6 +73,10 @@ class BernoulliRBM(BaseEstimator, TransformerMixin):
         Weight matrix, where n_features in the number of
         visible units and n_components is the number of hidden units.
 
+    h_samples_ : array-like, shape (batch_size, n_components)
+        Model matrix, where batch_size in the number of examples per minibatch
```

Comment:
I think that's not a very clear description.

Reply:
I am looking at the source code.
I think it has something to do with the hidden weights?

Would it be sufficient to say it is `Hidden Weight Matrix`